torchci/pages/api/job_annotation/[repoOwner]/[repoName]/annotations/[jobIds].ts

```diff
@@ -0,0 +1,32 @@
+import { getDynamoClient } from "lib/dynamo";
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const client = getDynamoClient();
+  const { jobIds, repoOwner, repoName } = req.query;
+  const jobIdsArr = JSON.parse(jobIds as string);
+  const queries = jobIdsArr.map((jobId: any) => {
+    return client.get({
+      TableName: "torchci-job-annotation",
```

Comment:
What's the tradeoff between using this `torchci-job-annotation` table instead of adding this data to the existing `torchci-workflow-job` table?

Reply:
I didn't really want to modify the items in the torchci-workflow-job table since to me they represent like a history of the events we get from Github (the schema matches theirs exactly). I don't want to be adding extra fields to it just for the sake of this feature. 

The main pro would be that we would be able to fetch all of the classifications directly if we put them in the same table but it's not that big of a deal IMO.

Reply:
It's more of a perf and code optimization, but yes, the tradeoff is you'd be modifying the "untouched historical record"

We can leave things as they are for now